RealismOverhaul.netkan

```diff
@@ -23,30 +23,29 @@
         { "name" : "B9-PWings-Fork" },
         { "name" : "ConnectedLivingSpace" },
         { "name" : "DeadlyReentry" },
-        { "name" : "HangarExtender" },
+        { "name" : "HangerExtenderExtended" },
         { "name" : "KSP-AVC" },
         { "name" : "MechJeb2" },
         { "name" : "ProceduralFairings" },
         { "name" : "ProceduralFairings-ForEverything" },
         { "name" : "ProceduralParts" },
         { "name" : "PersistentRotation" },
-        { "name" : "RCSBuildAid" },
+        { "name" : "RCSBuildAidCont" },
         { "name" : "RealSolarSystem" },
         { "name" : "KSCSwitcher" },
         { "name" : "RealismOverhaulCraftFiles" },
         { "name" : "RemoteTech" },
         { "name" : "SemiSaturatableRW" },
         { "name" : "TACLS" },
         { "name" : "TestFlight" },
-        { "name" : "TextureReplacer" },
+        { "name" : "TextureReplacerReplaced" },
         { "name" : "Toolbar" },
         { "name" : "VenStockRevamp"}
     ],
     "suggests"   : [
         { "name" : "DMagicOrbitalScience" },
         { "name" : "KAS" },
         { "name" : "SCANsat" },
-        { "name" : "AIESAerospace-Unofficial" },
         { "name" : "ALCOR" },
         { "name" : "ATKPropulsionPack" },
         { "name" : "AtomicAge" },
```

Comment:
Not updated since 1.0.5 but compatibility set to 1.1.3

Reply:
removed